R/dm.R

```diff
@@ -485,6 +480,29 @@ print.dm <- function(x, ...) {
   invisible(x)
 }
 
+print.zoomed_dm <- function(x, ..., n = NULL, width = NULL, n_extra = NULL) {
+
+  df <- get_zoomed_tbl(x)
```

Comment:
This part up to and including the `format()` call inside `cat_line()` can be part of a new `format.zoomed_dm()` .